src/View/Item/navBar.html.twig

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
```

Comment:
What is the point of this page ? Is it still usefull, cause I see there are still your problems with "assets/..." instead of "/assets/..."

Reply:
delete file navBar.html.twig and now navbar present on layout.html.twig only.